### ISSUE TITLE
Legacy behavior is broken

### ISSUE THREAD

**Issue by joshuaauerbachwatson** [nimbella/nimbella-sdk-nodejs#5]
**Describe the bug**
The SDK is supposed to support both the new `storageClient` function (which returns an abstract handle) and the legacy `storage` function (which only works on GCS and returns a `Bucket` object).   A recent change caused the legacy behavior to stop working ... the breakage is at this point: https://github.com/nimbella/nimbella-sdk-nodejs/blob/2c0875917c97682189ff6aa0c12420ead93759a2/index.js#L76.  

**To Reproduce**
```
    const nim = require('@nimbella/sdk')
    const bucket = await nim.storage()
```

**Expected behavior**
The variable `bucket` is initialized with a `Bucket` object as defined by the Google storage client.

**Comment by jthomas** [comment]
This bug should be fixed in v1.3.1

**Comment by joshuaauerbachwatson** [comment]
I think this has been fixed but I wait for a final agreement from @jthomas before closing.
